tests/conftest.py

```diff
@@ -197,6 +197,12 @@ def time_series_data():
     return {"np3d": np3d, "np2d": np2d, "df3d": df3d, "df2d": df2d, "nested": nested}
 
 
+@pytest.fixture()
+def minmax_scale_data():
+    x = [[2, 1, 3, 9], [4, 3, 4, 0], [6, 7, 2, 2]]
```

Comment:
It is probably easier and clearer to set this to be a DataFrame here, rather than converting in each individual unit test